### PR TITLE
added PG workaround with mount point

### PR DESCRIPTION
**Pull request by mustafa-travisci**
Also suggested as in the Zendesk ticket:

https://travisci.zendesk.com/agent/tickets/19064

### CODE REVIEW

**Comment by mustafa-travisci** [comment]
#2827

**Comment by mustafa-travisci** [comment]
travis-ci/travis-ci#9036

File: user/trusty-to-xenial-migration-guide.md [hunk 1]
```diff
@@ -26,6 +26,18 @@ Services like [MySQL or PostgreSQL](https://docs.travis-ci.com/user/database-set
     - mysql
 ```
 
+When you are trying to run PostgreSQL and you are facing a memory problem i.e. running `sudo df -h` and getting output from the mount points;
```
Comment (Pezi777) [inline user/trusty-to-xenial-migration-guide.md:29]:
```suggestion
When you are trying to run PostgreSQL and you are facing a memory problem, i.e. running `sudo df -h` and getting output from the mount points;
```

File: user/trusty-to-xenial-migration-guide.md [hunk 2]
```diff
@@ -26,6 +26,18 @@ Services like [MySQL or PostgreSQL](https://docs.travis-ci.com/user/database-set
...
+
+```
+none            768M  768M     0 100% /var/ramfs
+```
+
+try to add the following line of code into your `.travis.yml`
```
Comment (Pezi777) [inline user/trusty-to-xenial-migration-guide.md:35]:
```suggestion
try to add the following line of code to your `.travis.yml`:
```

**Pull request merged**